doc/source/whatsnew/v2.1.0.rst

```diff
@@ -275,6 +276,9 @@ Notable bug fixes
 
 These are bug fixes that might have notable behavior changes.
 
+-
+-
+
```

Comment:
This section is blank; I just wanted to make this more apparent.  cc @lithomas1

Reply:
My bad, I deleted the lines but not the section 🤦 .

Can we just take the whole section out?

Reply:
Sounds good! I was thinking we may still add something here, but that seems quite unlikely. Will remove in this PR.